code/modules/mob/living/living.dm

```diff
@@ -1375,3 +1375,9 @@
 	if(is_servant_of_ratvar(src) && !iseminence(src))
 		eminence.selected_mob = src
 		to_chat(eminence, "<span class='brass'>You select [src].</span>")
+
+/mob/living/proc/set_gender(ngender = NEUTER, silent = FALSE, update_icon = TRUE, forced = FALSE)
```

Comment:
Why does this not work on mobs with a ckey?

Reply:
Updated to make it work on mobs with Ckeys